docker-compose.override.yml

```diff
@@ -39,9 +39,11 @@ services:
       - .:/openlibrary
 
   db:
-    image: postgres:9.3
+    image: postgres:13.4
     networks:
       - dbnet
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
Can you add a comment about what this does/why it's necessary?

Reply:
Well , it is a requirement for the new image